src/utils.js

```diff
@@ -1,4 +1,5 @@
 const core = require("@actions/core");
+const escapeRegExp = require("lodash.escaperegexp");
```

Comment:
@iamdharmesh Curious for your thoughts here. I'm including this here and also installed the dependency in `package.json` but it doesn't seem like it's being included in the built file `dist/index.js`, unless I'm missing it? Wondering if there's something I need to do differently to get this dependency included?

Reply:
Never mind, I believe I've fixed this by installing just the required package instead of `lodash`